### PR TITLE
People 22, Round 1

### PR DESCRIPTION
**Pull request by a-wagner**
Still haven't received info from everyone, but I suppose I'll add people in rounds as their info becomes available.

@dbp I moved you up to faculty, but do you want to change any of your info? Photo, etc.

### CODE REVIEW

**Comment by dbp** [comment]
Hmm. Maybe could switch to my "official" headshot? https://dbp.io/static/dbp.jpg

Thanks!

**Review by mhyee — APPROVED** [review]
Looks great, thanks! (Added @CosineP as a reviewer, as the official webmaster.)

**Comment by mhyee** [comment]
Oh wait, one comment---does Max's blank bio render without breaking anything?

File: people.rkt [hunk 1]
```diff
@@ -687,6 +725,10 @@
                         #:dissertation "http://ccs.neu.edu/home/types/resources/pdf/g-dissertation-2020.pdf"
                         #:extra "(Brown University)"
                         #:personal-site "http://ccs.neu.edu/home/types/")
+              @(alumnus "Daniel Patterson" #:year 2022
+                        #:dissertation "https://dbp.io/pubs/2022/dbp-dissertation.pdf"
```
Comment (bennn) [inline people.rkt:729]: Time for a blog post? (Leif too)

Comment (LeifAndersen) [reply people.rkt:729]: You could cross post: https://blog.visr.pl/posts/use-deps/

Comment (LeifAndersen) [reply people.rkt:729]: Oh...wait, I see what you were going for. nvm. :) But ya, I should probably be there...unless you want me to stay listed as a PhD student forever. :P )

Comment (a-wagner) [reply people.rkt:729]: Yes, sorry! I wasn't sure when alum status officially kicks in—I was just going based off your website. Do you have a preferred link do your dissertation, or should I use the [ProQuest](https://www.proquest.com/docview/2719410129/abstract/8E84388F01D348FDPQ/1?accountid=12826) one? I'm not sure if that link works for everyone.

Comment (dbp) [reply people.rkt:729]: Not @LeifAndersen, but apparently ProQuest watermarks if you don't have an institutional subscription :( (I found that out when sending stuff for the JFP dissertation thingy). Personal webpages have their issues, but at least they have the real damn thing...

Comment (dbp) [reply people.rkt:729]: > Time for a blog post? (Leif too)

Once a blog editor, always the blog editor

File: people.rkt [hunk 2]
```diff
@@ -242,6 +242,21 @@
                   #:img "steven_holtzen.png"]{
             My research focuses on programming languages and artificial intelligence. In particular, my goal is to use programming languages and program analysis techniques as a foundation for specifying and reasoning about probabilistic models. Towards this end I am interested in the design, implementation, and applications of probabilistic programming languages; foundations of probabilistic inference and tractable probabilistic modeling; automated reasoning; and probabilistic verification.
           }
+
+          @person[#:name "Daniel Patterson"
+                  #:title "Assistant Teaching Professor"
+                  #:e-mail "dbp@ccs.neu.edu"
```
Comment (dbp) [inline people.rkt:248]: Could you actually switch this to dbp@dbpmail.net ? I can't send as it, so it's somewhat annoying to receive mail to. (and somewhat out of principle, don't really want to put @northeastern.edu one)

**Review by dbp — APPROVED** [review]

File: people.rkt (hunk 2 above)
Comment (a-wagner) [reply people.rkt]: Done

File: people.rkt [hunk 3]
```diff
@@ -242,6 +242,21 @@
...
           }
+
+          @person[#:name "Daniel Patterson"
+                  #:title "Assistant Teaching Professor"
+                  #:e-mail "dbp@dbpmail.net"
+                  #:website "https://dbp.io"
+                  #:history @list[
+
+                  "Joined Northeastern Faculty, 2022"
+                  "Joined Northeastern, 2016"
```
Comment (CosineP) [inline people.rkt:253]: Should this follow the usual PhD, Northeastern University, 2022 sort of format?

**Review by CosineP — APPROVED** [review]
This all looks good to me!    Moving Leif to alums was mentioned as well I think?
Sorry I still haven't gotten back with my own information - it might be a bit if it's alright to join with "Round 2."
Happy to merge this whenever and make sure it goes up, just lmk when you feel it's "ready" as round 1.

**Comment by dbp** [comment]
Some random notes:

Ben Lerner's title should be fixed. He's not a lecturer (hasn't been for years).

Also, Aaron should me moved to alums. (MS, 2022)

File: people.rkt (hunk 3 above)
Comment (a-wagner) [reply people.rkt]: Done

**Comment by CosineP** [comment]
Oops!   i should have caught this.   Looks like the merge with this commit[1] went bad; now we have two listings of Jason Hemann in Alumni

[1] https://github.com/nuprl/website/commit/b9ff34727274e6ca3a91fa750ad466edfff16ef8

**Comment by mhyee** [comment]
Sorry, I didn't catch this earlier, but Oli graduated in June last year.

**Comment by julbinb** [comment]
Jason is listed twice in the former faculty

**Comment by a-wagner** [comment]
@CosineP @mhyee @julbinb see #322.

**Pull request merged**